src/app/query-home/results/index.tsx

```diff
@@ -5,33 +5,31 @@ import {useResultsData} from "./data-hook"
 import {MainInspector} from "./main-inspector"
 import {ResultsError} from "./results-error"
 import ResultsTable from "./results-table"
-import * as Styled from "./results.styled"
 import {useResultsView} from "./view-hook"
 import Results from "src/js/state/Results"
+import styled from "styled-components"
 import {MAIN_RESULTS} from "src/js/state/Results/types"
 
+const BG = styled.div`
+  display: flex;
+  flex-direction: column;
+  border: none;
+  position: relative;
+  flex: 1;
+`
+
+const Body = styled.div`
+  flex: 1;
```

Comment:
Adding a comment so I can link to it from https://github.com/brimdata/brim/pull/2513.